portal-api/src/v1/users/controller.js

```diff
@@ -3,11 +3,12 @@ const now = require('../../now');
 const db = require('../../drivers/db-client');
 const sendEmail = require('../email');
 const businessRules = require('../../config/businessRules');
-const { BLOCKED, ACTIVE } = require('../../constants/user').DEAL_STATUS;
 const { sanitizeUser } = require('./sanitizeUserData');
 const utils = require('../../crypto/utils');
 const CONSTANTS = require('../../constants');
 const { isValidEmail } = require('../../utils/string');
+const { USER, PAYLOAD } = require('../../constants');
+const payloadVerification = require('../helpers/payload');
```

Comment:
Should this helper be called `payloadVerification.js` ?

Reply:
I am happy to leave it as `payload`.